lucene/core/src/test/org/apache/lucene/util/bkd/TestBKD.java

```
@@ -1698,6 +1698,16 @@ public void getValue(int i, BytesRef packedValue) {
           public byte getByteAt(int i, int k) {
             throw new UnsupportedOperationException();
           }
+
+          @Override
+          public void assign(int from, int to) {
+            // do nothing
+          }
+
+          @Override
+          public void finalizeAssign(int from, int to) {
+            // do nothing
+          }
```

can you throw an UnexpectedOperationException in both these methods instead, like we do for other methods?

---

comment addressed.